src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/TovStar.hpp

```diff
@@ -296,10 +326,17 @@ class TovStar : public virtual evolution::initial_data::InitialData,
     static type lower_bound() { return 1.; }
   };
 
+  /// Areal (Schwarzschild) or isotropic coordinates
```

Comment:
add to the class documentation what the spatial metric potentials are in isotropic coordinates